### ISSUE TITLE
nvidia docker runtime error install on RHEL8.2 for aarch64

### ISSUE THREAD

**Issue by mrmeswani** [NVIDIA/nvidia-docker#1361]
### 1. Issue or feature description
Error trying to install nvidia-docker runtime on RHEL8.2 for aarch64
Error is:
$ sudo yum install -y nvidia-docker2
Errors during downloading metadata for repository 'nvidia-container-runtime':
  - Status code: 404 for https://nvidia.github.io/nvidia-container-runtime/centos7/aarch64/repodata/repomd.xml (IP: 185.199.108.153)
Error: Failed to download metadata for repo 'nvidia-container-runtime': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried

### 2. Steps to reproduce the issue
Try to install nvidia-docker on aarch64 RHEL based machine 

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)
Some extra info 
$ uname -a
4.18.0-193.el8.aarch64

$ docker version 

Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.39
 Go version:        go1.13.10
 Git commit:        48a6621
 Built:             Mon Jun 22 15:47:41 2020
 OS/Arch:           linux/arm64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.1
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       4c52b90
  Built:            Wed Jan  9 19:06:23 2019
  OS/Arch:          linux/arm64
  Experimental:     false

**Comment by refactormyself** [comment]
Have you tried the commands given on the [ReadMe of this repo](https://github.com/NVIDIA/nvidia-docker#centos-7x8x-docker-ce-rhel-7x8x-docker-ce-amazon-linux-12)?
I see these specifically for the RedHat family, it works for even on an old Fermi GPU.
```
distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.repo | sudo tee /etc/yum.repos.d/nvidia-docker.repo

sudo yum install -y nvidia-container-toolkit
sudo systemctl restart docker
```

**Comment by mrmeswani** [comment]
it seems that the issue was the test was failing (see below) but the docker and nvidia container toolkit was correctly installed. 

docker run --gpus all nvidia/cuda:10.0-base nvidia-smi  ## this test always failed.  Also tried other cuda versions including 11.0 which was the version on bare metal. 

I tried the test with "docker run -it --rm --gpus all ubuntu nvidia-smi"  and it works fine and I was able to run GPU dockers for ARM. 

I would suggest to check why the cuda:XX-base containers are not working, but my issue is resolved.
